packages/sdk/test/methods.spec.ts

```diff
@@ -182,6 +206,59 @@ describe('LookerNodeSDK', () => {
     await sdk.authSession.logout()
   }
 
+  describe('downloads', () => {
+    it('png and svg', async () => {
+      const sdk = new LookerSDK(session)
+      const looks = await sdk.ok(sdk.search_looks({ limit: 1 }))
+      let type = ''
+      let id = ''
+      expect(looks).toBeDefined()
+      if (looks.length > 0) {
+        type = 'look'
+        id = looks[0].id!.toString(10)
+      } else {
+        const dashboards = await sdk.ok(sdk.search_dashboards({ limit: 1 }))
+        expect(dashboards).toBeDefined()
+        if (dashboards.length > 0) {
+          type = 'dashboards'
+          id = dashboards[0].id!
+        }
+      }
+      expect(type).toBeDefined()
+      expect(id).toBeDefined()
+      const image = await sdk.ok(
+        sdk.content_thumbnail({ type: type, resource_id: id, format: 'png' })
+      )
+      expect(image).toBeDefined()
+      expect(mimeType(image)).toEqual('image/png')
+      const svg = await sdk.ok(
+        sdk.content_thumbnail({ type: type, resource_id: id, format: 'svg' })
+      )
+      expect(svg).toBeDefined()
+      expect(svg).toMatch(/^<\?xml/)
+    })
+  })
+
+  describe('PUT smoke test', () => {
+    it('set default color collection', async () => {
+      const sdk = new LookerSDK(session)
+      const current = await sdk.ok(sdk.default_color_collection())
+      expect(current).toBeDefined()
+      const cols = await sdk.ok(sdk.all_color_collections())
+      const other = cols.find((c) => c.id !== current.id)
+      expect(other).toBeDefined()
+      // tests to stop lint from complaining
+      if (other && other.id && current.id) {
+        const actual = await sdk.ok(sdk.set_default_color_collection(other.id))
+        expect(actual).toBeDefined()
+        expect(actual.id).toEqual(other.id)
+        const updated = await sdk.ok(sdk.default_color_collection())
+        expect(updated.id).toEqual(actual.id)
+        await sdk.ok(sdk.set_default_color_collection(current.id))
```

Comment:
no follow on assertions about the result of this line?

Reply:
I suppose I could, but it seems the only failure condition you might have at that point would be if the server goes away before or during that last "reset" call, and if that happens, you'll have other issues to deal with anyway?